workflow-job

```diff
@@ -447,10 +444,8 @@ flows["${sha1}"] = {
             sh "cd dockerfiles/run/guice/cassandra-ldap; docker build --tag=${images.jamesCassandraLdap} ."
 
             dockeringGuiceCassandraLdap.success()
-        }
 
-        stage "Deploying Guice Server with JPA then provision data ${buildId}"
-        node {
+            //Deploying Guice Server with JPA then provision data
```

Comment:
Why do you remove all stage ?

Reply:
because we can only launch stages on the slaves, so we put every steps into only one stage (and set it to a slave labeled as 'remote')